src/components/characters/Characters.js

```diff
@@ -0,0 +1,56 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Character from './Character';
+import { getCharacters } from '../../services/rickAndMortyApi';
+import { withPaging } from '../paging/Paging';
+import styles from './Characters.css';
+
+class Characters extends PureComponent {
+  static propTypes = {
+    page: PropTypes.number,
+    updateTotalPages: PropTypes.func.isRequired
+  };
+
+  static defaultProps = {
+    page: 1
+  }
+
+  state = {
+    characters: []
```

Comment:
```suggestion
    characters: [],
    loading: false
```
To implement a loading feature, set some loading state to false and then toggle it's value in fetch characters (see below). Lastly, conditionally render a Loading component in your JSX.